microsoft/testsuites/vm_extensions/vmsnapshot_extension.py

```diff
@@ -100,6 +118,45 @@ def verify_vmsnapshot_extension(
                     pass
                 else:
                     raise e
-            time.sleep(1)
+            time.sleep(60)
             count = count + 1
         assert_that(count, "Restore point creation failed.").is_less_than(10)
+
+    def get_restore_point(
+        self,
+        log: Logger,
+        environment: Environment,
+        resource_group_name: str,
+        restore_point_collection: str,
+        restore_point: str,
+    ) -> int:
+        assert environment.platform
+        platform: AzurePlatform = environment.platform  # type: ignore
+        assert isinstance(platform, AzurePlatform)
+        client = get_compute_client(platform)
+        attempts = 0
+        max_attempts = 15
```

Comment:
use `retry` package, which has a lot of examples in LISA code base.

Reply:
Modified the function itself to raise errors instead of returning the value.
and upon checking the retry package as far as i understood, we are retrying the execution of a function that resulted in an error.
As i am raising 2 different errors, and for one i don't want to retry and for the other i wish to retry so avoiding it here.

Reply:
You can specify an error/exception to retry, other exceptions will be raised at first time.